R/metrics-quantile.R

```diff
@@ -532,10 +560,15 @@ bias_quantile_single_vector <- function(observed, predicted,
 ae_median_quantile <- function(observed, predicted, quantile_level) {
   assert_input_quantile(observed, predicted, quantile_level)
   if (!any(quantile_level == 0.5)) {
-    warning(
-      "in order to compute the absolute error of the median, `0.5` must be ",
-      "among the quantile levels given. Returning `NA`."
+    #nolint start: keyword_quote_linter
+    cli_warn(
+      c(
+        "x" = "In order to compute the absolute error of the median,
+        {.var 0.5} must be among the quantiles given.",
```

Comment:
is there a difference between `.var` and `.val`? Should this be `.val` here?

Reply:
You're right. It should be `.val` for values and `.var` for variables.